seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1160,6 +1534,7 @@
         "<code>id</code> attributes should be unique. Browsers won't enforce this, but it is a widely agreed upon best practice. So please don't give more than one element the same <code>id</code> attribute.",
```

Comment:
The description in this challenge is pretty similar to the next one. What if we change this line:
>There are several benefits to using id attributes, and you'll learn more about them once you start using jQuery.

to:
>There are several benefits to using id attributes: You can use an <code>id</code> to style a single element and later you'll learn that you can use them to select and modify specific elements with JavaScript.